evap/evaluation/models_logging.py

```diff
@@ -110,10 +111,12 @@ class Meta:
     @property
     def field_context_data(self):
         model = self.content_type.model_class()
-        return {
-            field_name: list(_field_actions_for_field(model._meta.get_field(field_name), actions))
-            for field_name, actions in self.data.items()
-        }
+        model_field_names = {f.name for f in model._meta.get_fields()}
+        context_data = {}
+        for field_name, actions in self.data.items():
+            if field_name in model_field_names:
+                context_data[field_name] = list(_field_actions_for_field(model._meta.get_field(field_name), actions))
+        return context_data
```

Comment:
You can keep this as a list comprehension, they support nested-ifs:
```python3
return {
    field_name: list(_field_actions_for_field(model._meta.get_field(field_name), actions))
    for for field_name, actions in self.data.items()
    if field_name in model_field_names
}
```

looks more complex at first, but makes it clear that all we're doing is creating a dictionary and returning that, so as a reader, you don't have think about so much control flow and state manipulation.